x-pack/plugins/fleet/common/openapi/paths/setup_preconfiguration.yaml

```diff
@@ -0,0 +1,22 @@
+put:
+  summary: Preconfiguration
+  tags: []
+  responses:
```

Comment:
I think the request body is missing from here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/routes/preconfiguration/index.ts#L28

Reply:
I just added it :)